### PR TITLE
Extract sass compilation tests out of obt and into a separate tool

### PR DESCRIPTION
**Pull request by JakeChampion**
The only testing task remaining in obt is the JavaScript tests

### CODE REVIEW

File: .github/workflows/test-tools-sass-compilation.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: volta-cli/action@v1
```
Comment (notlee) [inline .github/workflows/test-tools-sass-compilation.yml:15]: cool

**Comment by notlee** [comment]
Looks good 👍 need to also add it to the component's test workflow?

**Comment by JakeChampion** [comment]
> Looks good 👍 need to also add it to the component's test workflow?

That's done via the test.bash file :D

**Comment by notlee** [comment]
Oh I missed that, that's new!

**Comment by notlee** [comment]
oh no 😢 
Screenshot 2021-11-04 at 17 20 19

**Comment by JakeChampion** [comment]
> oh no 😢 Screenshot 2021-11-04 at 17 20 19

Yay! I'll fix that tomorrow 🥳

**Comment by notlee** [comment]
👀 https://github.com/ricklamers/sass-bin/blob/1cebdf30a1815efa38731fd92627bd5a48c5afa5/src/sass#L16

**Comment by JakeChampion** [comment]
> 👀 [ricklamers/sass-bin@`1cebdf3`/src/sass#L16](https://github.com/ricklamers/sass-bin/blob/1cebdf30a1815efa38731fd92627bd5a48c5afa5/src/sass#L16)

https://github.com/sass/dart-sass/issues/1125

**Comment by JakeChampion** [comment]
> > 👀 [ricklamers/sass-bin@`1cebdf3`/src/sass#L16](https://github.com/ricklamers/sass-bin/blob/1cebdf30a1815efa38731fd92627bd5a48c5afa5/src/sass#L16)
> 
> [sass/dart-sass#1125](https://github.com/sass/dart-sass/issues/1125)

https://github.com/ricklamers/sass-bin/pull/1

**Comment by JakeChampion** [comment]
@notlee this should now work on your m1 mac 👍

**Comment by notlee** [comment]
Unfortunately not @JakeChampion! 😿 

```
❯ npm exec -w components/o-autocomplete -c "npm run test"

> @financial-times/o-autocomplete@1.4.1 test
> bash ../../scripts/component/test.bash

Testing compilation for the core brand without calling any of the Sass API.
Error: Cannot find module 'sass-bin-darwin-64/bin/sass'
Require stack:
- /Users/leemoody/Projects/financial-times/origami/node_modules/sass-bin/src/sass
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.resolve (node:internal/modules/cjs/helpers:100:19)
    at generateBinPath (/Users/leemoody/Projects/financial-times/origami/node_modules/sass-bin/src/sass:48:23)
    at Object.<anonymous> (/Users/leemoody/Projects/financial-times/origami/node_modules/sass-bin/src/sass:83:3)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:79:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/Users/leemoody/Projects/financial-times/origami/node_modules/sass-bin/src/sass'
  ]
}
/Users/leemoody/Projects/financial-times/origami/node_modules/sass-bin/src/sass:51
    throw new Error(`The package "${pkg}" could not be found, and is needed by sass-bin.
    ^
```

**Comment by notlee** [comment]
I noticed the readme has been updated to recommend running `arch -x86_64 zsh -c "npm install sass-bin"` and that seems to install successfully

**Review by notlee — APPROVED** [review]
Thanks for your sass-bin work :) This is working now, with the exception of whatever is making o-autocomplete fail

**Pull request merged**